chapters/executors.rst

```diff
@@ -1,6 +1,6 @@
-================================================
-Understanding Jina Executors and How to Use Them
-================================================
+==================================
+Jina Executors and How to Use Them
```

Comment:
We should use title case here too

Reply:
According to https://titlecase.com/ it is title case